x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/job/process/autodetect/state/ModelSizeStats.java

```diff
@@ -262,6 +304,8 @@ public boolean equals(Object other) {
 
         private final String jobId;
         private long modelBytes;
+        private long modelBytesExceeded;
+        private long modelBytesMemoryLimit;
```

Comment:
These two need to be `Long`, otherwise they'll default to `0` and zeroes will then propagate into objects that should really contain `null`s.